sailor/assetcentral/equipment.py

```diff
@@ -80,6 +80,8 @@
     _AssetcentralField('_class', 'class'),
 ]
 
+timeout_seconds = sap_iot.fetch.GET_INDICATOR_DATA_TIMELIMIT_SECONDS
```

Comment:
I don't actually think we need this constant, just a fixed default of `None` in the function headers would be fine for me. 
As it is I had to first jump here to figure out the definition of `timeout_seconds`, and then jump to fetch to figure out the definition of `sap_iot.fetch.GET_INDICATOR_DATA_TIMELIMIT_SECONDS` before I knew the default was `None`.

Reply:
Done